workflows/virtual-screening-blueprint/definition/main.nf

```diff
@@ -29,7 +27,6 @@ workflow {
     )
 
     smiles = Channel.fromPath(smi_input_path)
```

Comment:
is this going to generate a new task per smiles string? if so, I think this will scale out too far

Reply:
it will generate a new task per SMI file, and there could be multiple SMILES in the file.